sop-core/src/Data/SOP/Constraint.hs

```diff
@@ -13,6 +13,7 @@ module Data.SOP.Constraint
   ) where
 
 import Data.Coerce
+import Data.Kind (Type)
 import GHC.Exts (Constraint)
```

Comment:
Probably best to import `Constraint` from `Data.Kind` as well now, rather than from `GHC.Exts`.